tests/cypress/e2e/filters-data.frame.cy.js

```diff
@@ -0,0 +1,150 @@
+describe("data.frame filters", () => {
```

Comment:
So for cypress tests you assume that there is a specific class id/name, so that's why you put all those `divs(` around in R code. Then you need to have `node` and know JavaScript to write tests? Is that right? Is JS part of CoreDev regular skillset?